<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GOTCHA</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="import_data">
       <attribute name="title">
        <string>Import Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="2">
         <widget class="QLabel" name="features_label">
          <property name="text">
           <string>No features file selected.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="log_label">
          <property name="text">
           <string>No log file selected.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="log_button">
          <property name="text">
           <string>Select Log File</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="features_button">
          <property name="text">
           <string>Select Features File</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="5">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="2" colspan="3">
         <widget class="QPushButton" name="read_button">
          <property name="text">
           <string>Read Log File</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="6">
         <widget class="QTableView" name="data_table"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="search_data">
       <attribute name="title">
        <string>Search Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="6" column="0">
         <widget class="QPushButton" name="search_button">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QTableView" name="search_table">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="export_results_button">
          <property name="text">
           <string>Export Results</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QTextEdit" name="search_textedit">
          <property name="placeholderText">
           <string>Enter Search Term</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="search_result_label">
          <property name="text">
           <string>Search Result Remarks: None</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="export_stats_button"/>
    <addaction name="help_button"/>
    <addaction name="about_button"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="export_stats_button">
   <property name="text">
    <string>Export Statistics</string>
   </property>
  </action>
  <action name="help_button">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="about_button">
   <property name="text">
    <string>About GOTCHA</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
